Dockerfile

```
@@ -141,23 +114,59 @@
-
-# Copy runtime scripts and set executable permissions
-COPY scripts/start_with_db.sh scripts/db_operations.sh /app/bin/
-RUN chmod +x /app/bin/*.sh
-
-COPY scripts/validate_and_start.sh /app/bin/validate_and_start.sh
-RUN chmod +x /app/bin/validate_and_start.sh
+# Create start.sh script directly in the container
+RUN mkdir -p /app/bin && \
+    cat > /app/bin/start.sh << 'EOF'
+#!/bin/bash
+set -e
```

Hadolint (2.12.0)

[error] unexpected 's'
expecting '#', '\', ADD, ARG, CMD, COPY, ENTRYPOINT, ENV, EXPOSE, FROM, HEALTHCHECK, LABEL, MAINTAINER, ONBUILD, RUN, SHELL, STOPSIGNAL, USER, VOLUME, WORKDIR, a pragma, at least one space, or end of input

(DL1000)